### DISCUSSION TITLE
The Argo-Rollouts fast rollback does not work when using Argo-Rollout and ArgoCD at the same time

### DISCUSSION THREAD

**Discussion by s2504s** [argoproj/argo-rollouts#2905] [General]
Hello, all!
After Argo-Rollout performed the fast rollback, ArgoCD syncs the Rollout resource and returns the previous state back
Is it default behavior? 
If yes, could you please clarify, how to prevent the syncing procedure for ArgoCD after making the Argo-Rollouts fast rollback

Thanks in advance!

**Comment by kostis-codefresh** [comment]
Could you describe a bit your setup? 

Normally this shouldn't happen https://argo-rollouts.readthedocs.io/en/latest/FAQ/#if-i-use-both-argo-rollouts-and-argo-cd-wouldnt-i-have-an-endless-loop-in-the-case-of-a-rollback
